definition/definition_ser_test.go

```diff
@@ -27,6 +24,11 @@ const defJSON = `
 	"tasks": [
 	{
 	  "id":"LogStart",
+		"settings" :{
```

Comment:
this doesn't make sense here.  these values only make sense for a loop.  ANd I would use the new loop configuration not the old stuff